### PR TITLE
python,cmake: update version script to support rc

### PR DESCRIPTION
**Pull request by vicentebolea**

### CODE REVIEW

**Comment by vicentebolea** [comment]
@anagainaru this will fix the error that you are observing in your PR

**Review by anagainaru — DISMISSED** [review]
I am not the best to review this so maybe wait for Scott to review (I'll approve in case you are confident in this). I'll rebase my PR once this is merged

**Comment by vicentebolea** [comment]
Scott is off until thursday, you can merge this and have it reverted if it causes trouble

**Review by anagainaru — APPROVED** [review]

**Pull request merged**
